packages/tools/pie-css/scss/mixins/_fontSize.scss

```diff
@@ -11,8 +11,11 @@
 //   @include font-size(--font-size-small);
 // }
 // -
-// Outputs: .text { font-size: calc(var(--font-size-small) * 1px); }
-// -
+// Outputs:
+// .text {
+//   font-size: calc(var(--font-size-small) * 1px);
+// }
+// ----------------------------------------------
 @mixin font-size($token) {
     font-size: calc(var($token) * 1px);
 }
```

Comment:
since both the mixin and function implement the same logic, when should we use each of them?

Reply:
The function is for when we want to use font-size for assignment (like in variables):

```scss
--my-font-size: #{p.font-size(--dt-var)};

// will give you
// --my-font-size: calc(…);
```

The mixin is if you want the output to include the left-hand-side `font-size:` as well:
```scss
@include font-size(--dt-var);

// will give you
// font-size: calc(…);
```

Reply:
makes sense 💯